README.md

```diff
@@ -15,3 +15,8 @@ Bitcoin-specific address encoding is handled by the `bitcoin-bech32` crate.
 
 The minimum supported Rust version with the standard library is **1.41.1**.
 
+
+## Benchmarks
+
+We use a custom Rust compiler configuration conditional to guard the bench mark code. To run the
+bench marks use: `RUSTFLAGS='--cfg=bench' cargo +nightly bench`.
```

Comment:
In 692a12c41078a497c01e3d7ea499b7533205f80d:

I think "benchmark" should be one word (it's split up in two locations).

Reply:
Oh yeah, good point.